### PR TITLE
Added intro and operations section to ch3

### PR DESCRIPTION
**Pull request by amandakube**

### CODE REVIEW

File: textbook/03/1/Operations.ipynb [hunk 1]
```diff
@@ -4,16 +4,470 @@
    "cell_type": "markdown",
```
Comment (campbelle1) [inline textbook/03/1/Operations.ipynb:4]: <p>Thirteen and five  -&gt;  thirteen and five</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (campbelle1) [inline textbook/03/1/Operations.ipynb:4]: <p>python  -&gt;  Python</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

**Comment by jesteria** [comment]
@amandakube This appears to have been superseded by #45 (right?).

**Comment by amandakube** [comment]
Yep!

**Pull request closed without merging**